packages/dependency-extraction-webpack-plugin/package.json

```diff
@@ -16,7 +16,7 @@
 	"bugs": {
 		"url": "https://github.com/woocommerce/woocommerce-admin/issues"
 	},
-	"main": "build/index.js",
+	"main": "src/index.js",
 	"module": "build-module/index.js",
 	"react-native": "src/index",
```

Comment:
This should surely be removed too, I don't think this is intended to be used in a react-native context so it might just add confusion.